cmd/state-installer/internal/installer/installation_darwin.go

```diff
@@ -2,7 +2,58 @@
 
 package installer
 
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/ActiveState/cli/internal/appinfo"
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/fileutils"
+	"github.com/gobuffalo/packr"
+)
+
+const appDir = "/Applications"
+
 // InstallSystemFiles installs files in the /Application directory
 func InstallSystemFiles(installDir string) error {
+	box := packr.NewBox("../../../../assets/macOS/app")
+	err := box.Walk(func(path string, _ packr.File) error {
+		if fileutils.IsDir(path) {
+			err := fileutils.Mkdir(filepath.Join(appDir, path))
+			if err != nil {
+				return errs.Wrap(err, "Could not create directory")
+			}
+		} else {
+			err := fileutils.WriteFile(filepath.Join(appDir, path), box.Bytes(path))
+			if err != nil {
+				return errs.Wrap(err, "Could not write icon file")
+			}
+		}
+		return nil
+	})
+	if err != nil {
+		return errs.Wrap(err, "Could not create application directory")
+	}
+
+	trayInfo := appinfo.TrayApp()
```

Comment:
So, the tray app is the only one, we need to symlink now?  That is good...

Reply:
Yes, `appinfo` takes care of that so we only need to like the `state-tray` binary